integration/containerd/cri/integration-tests.sh

```diff
@@ -72,6 +72,8 @@ ci_config() {
 		(
 		echo "Install cni config for cri-containerd test"
 		cd "${GOPATH}/src/${cri_containerd_repo}"
+		# Below script installing CNI config for containerd 1.3.0 installs ipv6 routes, so enable ipv6
+		sudo sysctl net.ipv6.conf.all.disable_ipv6=0
```

Comment:
I wonder if this should be part of a file, i,e https://github.com/kata-containers/tests/blob/master/.ci/install_kubernetes.sh#L81

Reply:
If this is purely for k8s, I agree. Could you also add an `info "enabling IPv6"` message too?

Reply:
Sure.